dockers/docker-orchagent/supervisord.conf.j2

```diff
@@ -288,3 +288,19 @@ dependent_startup_wait_for=swssconfig:exited
 environment=ASAN_OPTIONS="log_path=/var/log/asan/fdbsyncd-asan.log{{ asan_extra_options }}"
 {% endif %}
 {%- endif %}
+
+{% if DEVICE_METADATA.localhost.type %}
+{% if DEVICE_METADATA.localhost.type == "BmcMgmtToRRouter" %}
```

Comment:
Should we use a more general flag?
Such as add a `time_based_acl: enabled` in device metadata?